.travis.yml

```diff
@@ -29,3 +32,5 @@ after_success:
   # gover collects all .coverprofile files and saves it to one file gover.coverprofile
   - gover
   - goveralls -coverprofile=gover.coverprofile -service=travis-ci
+  # sync the docs only if everything else was successful
+  - cd script && ./sync-docs.sh
```

Comment:
nit:
How about -
`script/sync-docs.sh` in place of `cd script && ./sync-docs.sh`